books/free-programming-books-langs.md

```diff
@@ -1081,6 +1081,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [JavaScript ES6 and beyond](https://github.com/AlbertoMontalesi/JavaScript-es6-and-beyond-ebook) - Alberto Montalesi (PDF, epub)
 * [JavaScript For Cats](http://jsforcats.com) - Maxwell Ogden (HTML)
 * [JavaScript for Impatient Programmers (ES2020 edition)](https://exploringjs.com/impatient-js/toc.html) - Dr. Axel Rauschmayer (HTML)
+* [JavaScript for Impatient Programmers (ES2023 edition)](https://exploringjs.com/impatient-js/downloads/impatient-js-preview-book.pdf) - Axel Rauschmayer (PDF)
```

Comment:
PDF is a preview 50%. Please... use landing page which links all versions, including HTML having full content

https://exploringjs.com/impatient-js/